### ISSUE TITLE
[Accessibility]: Sentences split by links not read correctly

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#35193]
### Audit type

Screen reader navigation

### User journey audit issue

#33517, #33513, #33513

### Problem description

<img width="884" alt="image" src="https://user-images.githubusercontent.com/103087/167505991-4c805740-930c-47d1-af64-4df7d1672479.png">

Upload and index metadata often uses text split by several links (directory <link> indexed by <link> on <link> for <link>). When read by a screen reader, it only reads the text segments between links, or the focused link.

This header is another example:

<img width="640" alt="Screen Shot 2022-05-09 at 5 24 09 PM" src="https://user-images.githubusercontent.com/103087/167508749-5062e523-7add-430c-b4fc-4135c4b03493.png">

### Expected behavior

The sentence is readable without extreme effort.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
I think we can close this one, there's a lot of additional screen reader shortcuts that are intended to read text more naturally (e.g. [VoiceOver shortcuts](https://support.apple.com/en-gb/guide/voiceover/vo2706/mac)), and we probably shouldn't try to fight that too much.

I think for natural reading, a user would use something like read "From the VoiceOver cursor to the bottom". I tested this on this page and it worked well
